unicon_runner/executor/variants/sandbox/executor.py

```diff
@@ -0,0 +1,64 @@
+import asyncio
+import os
+
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+
+from unicon_runner.executor.variants.base import Executor, ExecutorResult
+from unicon_runner.schemas import Request, Status
+
+
+class SandboxExecutor(Executor):
+    """Uses conty"""
+
+    env = Environment(
+        loader=FileSystemLoader("unicon_runner/executor/variants/unsafe/templates"),
+        autoescape=select_autoescape(),
+    )
+    pyproject_template = env.get_template("pyproject.toml.jinja")
+    CODE_FOLDER_NAME = "src"
+    RUN_SCRIPT = "unicon_runner/executor/variants/unsafe/scripts/run.sh"
+    CONTY = os.getenv("CONTY_PATH")
+
+    async def _execute(self, request: Request, request_id: str, folder_path: str) -> ExecutorResult:
+        # 1. Copy the uv files
+        code_folder_path = os.path.join(folder_path, self.CODE_FOLDER_NAME)
+        os.mkdir(code_folder_path)
+
+        for file in request.files:
+            with open(os.path.join(code_folder_path, file.file_name), "w") as f:
+                f.write(file.content)
+
+        with open(os.path.join(folder_path, "pyproject.toml"), "w") as f:
+            pyproject_file = self.pyproject_template.render()
+            f.write(pyproject_file)
+
+        with open(os.path.join(folder_path, "README.md"), "w") as f:
```

Comment:
is there a use case for `README.md` here?

Reply:
not anymore, i originally encountered an error with README.md not existing without the `--no-project` parameter in the script, but encountered another weird error 

```
...The most likely cause of this is that there is no directory that matches the name of your project (src).\n\nAt least one file selection option must be defined in the `tool.hatch.build.targets.wheel` table, see: https://hatch.pypa.io/latest/config/build/\n\nAs an example, if you intend to ship a directory named `foo` that resides within a `src` directory located at the root of your project, you can define the following:\n\n[tool.hatch.build.targets.wheel]\npackages = [\"src/foo\"]\n---\n","status":"WA"}],"error":null}
```

uv run --no-project fixed it and somehow runs without it (so technically i don't need it) but now that i think about it i am not sure if that will affect me adding dependencies later ❓

Reply:
from the [docs](https://docs.astral.sh/uv/guides/scripts/#running-a-script-without-dependencies) `--no-project` seems to not install the dependencies listed in `pyproject.toml`

my understanding is that when `--no-project` is used, we are using the python interpreter under host's root `uv` virtual env

Reply:
the original error you faced is due to a missing `__init__.py` under `/src`. without `__init__.py`, `/src` is not recognised as a package hence the build error